ports/mimxrt/Makefile

```diff
@@ -282,6 +288,7 @@ SRC_QSTR += \
 	modmimxrt.c \
 	moduos.c \
 	pin.c \
+	fatfs_port.c \
```

Comment:
this doesn't need to be added here, it doesn't have any QSTR's in it

Reply:
fixed.